backend/src/Squidex.Infrastructure/Security/Extensions.cs

```diff
@@ -67,6 +67,12 @@ public static class Extensions
             return principal.Claims.FirstOrDefault(x => x.Type == OpenIdClaims.Email)?.Value;
         }
 
+        public static string? TryFindEmail(this ClaimsPrincipal principal)
```

Comment:
Can you not do this in the OpenIdHandler? In TokenValidated method...

Reply:
I checked and if you are using `options.GetClaimsFromUserInfoEndpoint` then the claims are not present in TokenValidated - they are fetched afterwards, so you need to handle it when searching the claims.